core/example/solvers/direct.h

```diff
@@ -0,0 +1,32 @@
+#ifndef TDS_TDS_INCLUDED
+#define TDS_TDS_INCLUDED
+
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_MemberTypes.hpp>
+
+#include <Kokkos_Core.hpp>
```

Comment:
Why do you include `<Kokkos_Core.hpp>` and the two Cabana headers?

Reply:
I think this is just a relic from a different build+linking approach that is obsolete. I've commented out quite a few instances of `#include <UnneededHeader.hpp>` throughout. I'll likely just remove those lines soon enough.

Reply:
Yep, removed those unused or redundant header includes.